_custombuild

```diff
@@ -91,6 +95,7 @@ var addon = gulp.src([
   // Fix the `gaia-icons` path after namespacing has completed.
   .pipe(replace(/customizer\-gaia\-icons\/customizer\-gaia\-icons\.css/g, 'gaia-icons/gaia-icons.css'))
   .pipe(wrapper({
+    header: 'var result;',
```

Comment:
I should probably put a comment here.. js-prettify doesn't agree with strict mode because of an undeclared `result` var.